pkg/annotations/annotations.go

```diff
@@ -35,4 +35,36 @@ const (
 	// UntrustedWorkload is the sandbox annotation for untrusted workload. Untrusted
 	// workload can only run on dedicated runtime for untrusted workload.
 	UntrustedWorkload = "io.kubernetes.cri.untrusted-workload"
+
+	vcAnnotationsPrefix = "com.github.containers.virtcontainers."
```

Comment:
can we move these to a new vc.go either in the same package or vendor it in?

Reply:
OK, I will vendored it in.